### ISSUE TITLE
Text on Playling List is cropped

### ISSUE THREAD

**Issue by HyperSB1** [mmuffins/mediamonkey-codemonkey#17]
Hi, this is my favourite MM skin but there's one thing that's always bothered me. The way that songs are rendered in the playing list makes text cropped off at the bottom, making letters with tails (and all those square brackets) a little annoying to read ('y' looks like a 'v'). It doesn't look like that in the previews but I'm not sure what I'm doing wrong. It remains cropped no matter what Display Size I use or what font I change it to. Is there any way for me to fix this or some kind of workaround I'm not aware of? There's not a great deal of forum help on MM sadly.
![text spacing issue](https://github.com/user-attachments/assets/f109ffb5-66eb-4af3-b949-38b062462fa9)

**Comment by mmuffins** [comment]
I have noticed that but have yet to find a good solution for it. It seems to be related to how row heights are calculated for various scale factors. The bigger the text scale, the less it becomes a problem, but everything I tried so far either breaks scaling or has some other side effects that are worse than the original problem.

**Comment by HyperSB1** [comment]
oh well that sucks, no worries though. in some of the screenshots I've seen it looks like the cropping is fixed, could that be because they're on a different OS? or aspect ratio? or monitor resolution?

**Comment by mmuffins** [comment]
It's mostly related to scaling, both for the application/desktop itself as well as the scaling setting in the skin. So basically if you select a bigger scaling level the issue gets better.

**Comment by HyperSB1** [comment]
Yep, when I change the skin's display size (that is what you mean by changing the scaling right? that's the only way I know how), it does reduce the cropping, but then makes the text too small to read comfortably. Where I have it at the moment is the best compromise for me in terms of readability. I did also try a different font (JetBrains Mono), but as expected it didn't help. Maybe if the font had a lot of bottom padding? but why would you design a font like that? By and large, I don't see a font change helping, but it's nice that I can.
